languages.toml

```diff
@@ -2091,6 +2091,20 @@ language-servers = [ "jsonnet-language-server" ]
 name = "jsonnet"
 source = { git = "https://github.com/sourcegraph/tree-sitter-jsonnet", rev = "0475a5017ad7dc84845d1d33187f2321abcb261d" }
 
+[[language]]
+name = "ada"
+scope = "source.ada"
+injection-regex = "ada"
+file-types = ["adb", "ads"]
+roots = ["alire.toml"]
+comment-token = "--"
+indent = { tab-width = 3, unit = " " }
+language-server = { command = "ada_language_server" }
```

Comment:
The format for declaring language servers changed after the 23.05 release. An `ada-language-server` key should be added under the `[language-server]` table at the top:

```toml
[language-server]
# ...
ada-language-server = { command = "ada_language_server" }
# ...

[[language]]
name = "ada"
language-servers = [ "ada-language-server" ]
# ...
```